Library/Homebrew/sorbet/custom_generators/env_config.rb

```diff
@@ -0,0 +1,46 @@
+# typed: true
+# frozen_string_literal: true
+
+require_relative "../../global"
+require_relative "../../env_config"
+
+def nilable?(method)
```

Comment:
This should be all without a `default` value, right? So I don't think we need this function.

Reply:
You're correct! Thanks.